markdown/org/docs/sewing/knit-binding/en.md

```diff
@@ -6,21 +6,22 @@ title: Knit binding
 
 Knit binding is a general term for a type of finish (typically) found on the neck or arm opening of T-shirts and other knitwear.
 
-There's two varieties of this finish, a knit band, and actual knit binding.
+There are two varieties of this finish, a knit _band_ and actual knit _binding_.
 
 ## Knit band
 
-![Like most T-shirts, the one this girl is wearing has a knit band at the neck opening.](knit_band.jpg)
+![Like most T-shirts, the one this woman is wearing has a knit band at the neck opening.](knit_band.jpg)
```

Comment:
There in only one person in the image, so it seems redundant to mention `woman` here (and also here https://github.com/freesewing/freesewing/pull/3842/files#diff-4fa783e1a15b6063eee9d93cd61d9c8d14b5e8190afc2469ad020bb412ba8f13R22)

```suggestion
![Like most T-shirts, this one has a knit band at the neck opening.](knit_band.jpg)
```